docs/general/server/media/media-segments.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
Would it make more sense to add this in the "metadata" category instead of "media"? Feels like "media" is to highlight the different media types (music/books etc) and folder structures

Reply:
I also don't really know where to put it besides it should be under server

Reply:
i agree